### ISSUE TITLE
in Work page view, hyperlink is missing discernible text

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#859]
On item page, hyperlink is missing discernible text. [Deque help page](https://dequeuniversity.com/rules/axe/4.3/link-name?application=AxeChrome ) for this issue. 

Page tested: https://ldr.lafayette.edu/concern/images/nv935360n 

Associated element/source code:
`<a href="/" id="top"></a>`

Selector:
`["iframe","#top"]`

**Comment by noraegloff** [comment]
Deque aXe tools severity level = serious

**Comment by rococodogs** [comment]
<img width="713" alt="Screen Shot 2022-03-16 at 8 35 57 AM" src="https://user-images.githubusercontent.com/2744987/158591288-4245ffce-b135-4aba-8210-badbe4e67d49.png">

i think this is being dynamically inserted by the universal viewer js code? (in the html template, that container div is entered as`<div id="uv" class="uv"></div>`.) annoying!
